.vscode/settings.json

```diff
@@ -0,0 +1,92 @@
+{
```

Comment:
vscode project files mixed with this PR: to remove (feel free to update .gitignore)